vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -218,7 +218,8 @@ function Install-WindowsPatches {
     # Windows Server 2019 update history can be found at https://support.microsoft.com/en-us/help/4464619
     # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
-    $patchUrls = @("http://download.windowsupdate.com/c/msdownload/update/software/updt/2020/10/windows10.0-kb4580390-x64_743bc31f33bf399c7f15ab020df685780faf4cb5.msu")
+    # 2020-11 Cumulative Update for Windows Server 2019 for x64-based Systems (KB4586793)
```

Comment:
Please also update the test. I have updated the wiki page.
https://github.com/Azure/AgentBaker/blob/202fef719a47568238d303b4ec4e3a497d6efe53/vhdbuilder/packer/test/vhd-content-test.ps1#L152

Reply:
Fixed test.